src/main/resources/liquibase/liquibase-deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+liquibase --driver=com.mysql.jdbc.Driver \
+     --classpath=${HOME}/.ivy2/cache/mysql/mysql-connector-java/jars/mysql-connector-java-5.1.35.jar \
+     --changeLogFile=src/main/resources/liquibase/changelog.xml \
+     --url="jdbc:mysql://localhost/thurloe_test" \
+     --username=travis \
+     --password="" \
+     migrate
```

Comment:
@kshakir do you know whether the liquibase lines in the application.conf make this redundant? Also if we add liquibase to the in-memory DB can we remove the "setupInMemoryDatabase" stuff?